LEAF_Request_Portal/templates/print_subindicators_ajax.tpl

```diff
@@ -1,21 +1,28 @@
 <!--{**}-->
         <!--{if $indicator.is_sensitive == 1}-->
             <div class="sensitiveIndicatorMaskToggle">
-                <input type="checkbox" id="sensitiveIndicatorMaskCheckbox_<!--{$indicator.indicatorID|strip_tags}-->_<!--{$indicator.series|strip_tags}-->" onClick="toggleSensitiveIndicator(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->, this.checked);">
-                <label for="sensitiveIndicatorMaskCheckbox_<!--{$indicator.indicatorID|strip_tags}-->_<!--{$indicator.series|strip_tags}-->" title="Show Sensitive Data" alt="Show Sensitive Data"  tabindex="0" onkeydown="if (event.keyCode==13){ this.click(); }"></label>
+                <input type="checkbox" id="sensitiveIndicatorMaskCheckbox_<!--{$indicator.indicatorID|strip_tags}-->_<!--{$indicator.series|strip_tags}-->" onClick=" if (stayVisible === 0) {stayVisible = 1;} else {stayVisible = 0;} toggleSensitiveIndicator(<!--{$indicator.indicatorID|strip_tags}-->, <!--{$indicator.series|strip_tags}-->, this.checked);">
```

Comment:
Inline Javascript like this should be avoided as much as possible. Instead, just create a function to handle toggling the `stayVisible` variable.

Also if the `stayVisible` variable is changed to a boolean, you could just say `stayVisible = !stayVisible` in the toggle function.

Reply:
Addressed in commit `af0`